internal/resource_mks_cluster/mks_cluster_resource_gen.go

```diff
@@ -1008,11 +1013,19 @@ func (v MetadataValue) String() string {
 func (v MetadataValue) ToObjectValue(ctx context.Context) (basetypes.ObjectValue, diag.Diagnostics) {
 	var diags diag.Diagnostics
 
-	annotationsVal, d := types.MapValue(types.StringType, v.Annotations.Elements())
```

Comment:
@venkatm-rafay Why are we changing annotations code here ?

Reply:
@chaitanya-rafay These are the generated files. I din't make made any changes.

Reply:
@vasu-rafay Can you check this ?

Reply:
@chaitanya-rafay Looks good. Additional logic to handle `Unknown` and `Null` values are introduced for `Map`. @venkatm-rafay Can we add `terraform-plugin-codegen-framework` to go dependencies ?